tap_shopify/streams/product_type.py

```diff
@@ -0,0 +1,80 @@
+import json
+import os
+import sys
+import shopify
+import singer
+from singer.utils import strftime, strptime_to_utc
+
+from tap_shopify.context import Context
+from tap_shopify.streams.base import Stream, shopify_error_handling
+
+LOGGER = singer.get_logger()
+
+
+class HiddenPrints:
+    def __enter__(self):
+        self._original_stdout = sys.stdout
+        sys.stdout = open(os.devnull, 'w')
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        sys.stdout.close()
+        sys.stdout = self._original_stdout
+
+
+class ProductType(Stream):
```

Comment:
Can you fix the name to be `ProductCategory`?

Reply:
on it!